doc/nrf/libraries/networking/nrf_cloud.rst

```diff
@@ -145,6 +145,26 @@ When the device receives the :c:enumerator:`NRF_CLOUD_EVT_FOTA_DONE` event, the
 The message payload of the :c:enumerator:`NRF_CLOUD_EVT_FOTA_DONE` event contains the :c:enum:`nrf_cloud_fota_type` value.
 If the value equals :c:enumerator:`NRF_CLOUD_FOTA_MODEM`, the application can optionally avoid a reboot by performing reinitialization of the modem and calling the :c:func:`nrf_cloud_modem_fota_completed` function.
 
+Building FOTA images
```

Comment:
This is rather generic stuff (except for the Cloud part of it) and I guess it could be better placed in, for example in the Getting started/Building and programming an application section (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_programming.html). The generic part could then be included here from the Building and programming an application. Opinions?